controllers/contactsController.js

```diff
@@ -3,7 +3,17 @@ const { Contact } = require("../models/contact");
 const { HttpError, ctrlWrapper } = require("../helpers");
 
 const listContacts = async (req, res) => {
-  const result = await Contact.find();
+  const { _id: owner } = req.user;
+  const { page = 1, limit = 20, favorite } = req.query;
+  const skip = (page - 1) * limit;
+  const result = await Contact.find(
+    { owner, ...(favorite !== undefined && { favorite }) },
```

Comment:
```suggestion
    { owner, ...(favorite !== undefined && { favorite }) },
``` 
нечитабельний чапис винесіть перевірку в окрему змінну. можна спростити favorite&& {favorite}